tidb-cloud/v7.1.0-performance-benchmarking-with-sysbench.md

```diff
@@ -0,0 +1,166 @@
+---
+title: TiDB Cloud Sysbench Performance Test Report
+summary: Introduce the Sysbench performance test results for TiDB Cloud.
+---
+
+# TiDB Cloud Sysbench Performance Test Report
+
+This document provides the Sysbench performance test steps and results for a TiDB Dedicated cluster with the TiDB version of v7.1.0. This report can also be used as a reference for the performance of TiDB v7.1.0 clusters.
+
+## Test overview
+
+This test aims at showing the Sysbench performance of TiDB v7.1.0 in the Online Transactional Processing (OLTP) scenario.
+
+## Test environment
+
+### TiDB cluster
+
+The test is conducted on a TiDB cluster with the following settings:
+
+- Cluster type: [Dedicated Tier](/tidb-cloud/select-cluster-tier.md#dedicated-tier)
+- Cluster version: v7.1.0
+- Cloud provider: AWS (us-west-2)
+- Cluster configuration:
+
+   | Node type | Node size | Node quantity | Node storage |
+   |:----------|:----------|:----------|:----------|
+   | TiDB      | 16 vCPU, 32 GiB | 2 | N/A |
+   | TiKV      | 16 vCPU, 64 GiB | 3 | 1000 GiB |
+
+
+### Parameter configuration
+
+   Enabling the below parameter can have a positive impact on performance in the 'oltp_insert','oltp_read_write','oltp_update_index','oltp_update_non_index' scenario.
```

Comment:
IMO, it would be better to say [log recycling](https://docs.pingcap.com/tidb/dev/tikv-configuration-file#enable-log-recycle-new-in-v630), which is enabled by default in v6.3.0, can improve the performance of these 4 scenarios. Enabling the below parameter can let the log recycling effective immediately after initialization, speeding up the benchmark. https://docs.pingcap.com/tidb/dev/tikv-configuration-file#prefill-for-recycle-new-in-v700